builds/linux/rhel/9/variables.pkr.hcl

```diff
@@ -10,8 +10,24 @@
 //  BLOCK: variable
 //  Defines the input variables.
 
+// Additional yum repositories
+variable "yum_repositories" {
```

Comment:
I don't see where this is being used for this enhancement.

Reply:
my bad, removed